doc/PluginTutorial.md

```diff
@@ -164,7 +164,7 @@ section:
 ```html
 	<head>
 		<!-- CSS -->
-		<link rel="stylesheet" href="../../css/roboto-condensed-min.css">
+		<link rel="stylesheet" href="../../css/roboto-condensed/css/roboto-condensed-min.css">
```

Comment:
I almost left the .css file where it was only adding the fonts folder but I ended up modeling it after font-awesome's directory structure, if preferred I can move the .css back to a simpler file path